client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -165,6 +166,18 @@
         this.edit = edit;
         this.editItem = null;
 
+        this.isDeskChanged = function (){
+            return desks.changeDesk;
+        }
+
+        this.highlightsDeskChanged = function (){
+            if(desks.changeDesk) {
+                $location.url('/workspace/monitoring');
```

Comment:
I think we can achieve the same using `superdesk.flags` here instead of `$location.url()`